src/main/java/com/alkemy/ong/web/controllers/ContactController.java

```diff
@@ -0,0 +1,62 @@
+package com.alkemy.ong.web.controllers;
+
+import com.alkemy.ong.domain.contact.Contact;
+import com.alkemy.ong.domain.contact.ContactService;
+import lombok.*;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/contacts")
+public class ContactController {
+    private final ContactService contactService;
+
+    public ContactController(ContactService contactService){
+        this.contactService = contactService;
+    }
+
+    @GetMapping()
+    public List<ContactDTO> getAllContacts() throws Exception {
+
+        List<ContactDTO> dto = new ArrayList();
+        List<Contact> contacts = contactService.getContacts();
+
+        if (!contacts.isEmpty()) {
+            for (Contact contact : contacts) {
```

Comment:
En lugar de for, usemos streams. 
La idea es convertir la lista de contacts a stream, transformarlos a dto y dsp agruparlos en una lista de dtos.
En pseudo-codigo seria algo asi:
contactService.getContacts().stream()
    .map(c -> toDto(c))
    .collect(toList());

El map opera sobre cada elemento de la lista. Entonces tendrías q tener un metodo toDto q haga la conversion de model a dto.
Y dsp el collect agrupa cada uno de los elementos nuevamente. En este caso, como necesitamos una lista, usamos el Collectors.toList()